dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/api/OpenTelemetryAccessor.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package io.openliberty.microprofile.telemetry.internal.interfaces;
+package io.openliberty.microprofile.telemetry.api;
```

Comment:
I think this api package needs to be its own project with an [appropriate name][naming] as it will need to be included as a jar in the dev directory, have its javadoc published and published on maven central.

[naming]: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines